<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Muhammads-MacBook-Air.local" failures="1" tests="6" name="Default test" time="16.871" errors="0" timestamp="2022-07-08T14:39:45 WIB">
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="5.273">
    <failure type="java.lang.AssertionError" message="expected [http://34.101.70.83:5100/product_category] but found [http://34.101.70.83:5100/product_category/add]">
      <![CDATA[java.lang.AssertionError: expected [http://34.101.70.83:5100/product_category] but found [http://34.101.70.83:5100/product_category/add]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at pages.AddProductCategory.linkcheck(AddProductCategory.java:88)
at pageSteps.AddProductCategorySteps.linkcheck(AddProductCategorySteps.java:59)
at âœ½.to validate add product category move to page(file:///Users/muhammadbaihaqy/eclipse-workspace/adminNufarm/src/main/java/features/addProductCategory.feature:14)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="2.144"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="2.119"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="2.234"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="2.205"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="2.646"/>
</testsuite> <!-- Default test -->
